manager.py

```diff
@@ -13,13 +13,21 @@
 import shutil
 import tempfile
 import multiprocessing
+import multiprocessing.pool
+import platform
 
+from manager.wasabi_clients.client_versions_enum import VersionsEnum
+from manager.wasabi_clients.wasabi_client_v1 import WasabiClientV1
+from manager.wasabi_clients.wasabi_client_v2_old import WasabiClientV2Old
+from manager.wasabi_clients.wasabi_client_v2_new import WasabiClientV2New
 
 BTC = 100_000_000
 SCENARIO = {
     "name": "default",
     "rounds": 10,  # the number of coinjoins after which the simulation stops (0 for no limit)
     "blocks": 0,  # the number of mined blocks after which the simulation stops (0 for no limit)
+    "default_version": "2.0.4",
+    "distributor_version": "2.0.4",
```

Comment:
```suggestion
```

Reply:
If excluded from both manager and genscen, it will be possible to change distributor version only manually. The idea is to set specific version of distributor for usage with GUI versions as defaults, as the distributor is not important for the CoinJoin itself and would waste resources. Is the removal to use the distributor version only if really needed?

Reply:
This could be resolved by adding an option to `genscen.py` to use specific distributor version in a scenario. Otherwise, I would side with having by default every client using the same version, so you do not need to pull/build extra images, even though in some cases it might waste some additional resources on the distributor.

I would eventually like to drop distributor from the setup entirely, as the initial distribution can be done directly by btc-node, but that is for another PR.